fix(versions): update Activiti/activiti-cloud-notifications-service-graphql versions

jx-activiti-cloud opened the pull request:
[UpdateBot](https://github.com/jenkins-x/updatebot) pushed maven dependency: `org.activiti.cloud.query:activiti-cloud-query-dependencies` to: `7.1.18`

Commit:
fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.18

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.query:activiti-cloud-query-dependencies 7.1.18

ryandawsonuk committed:
do we need separate ws schema? (#37)

Merging this to get everything building again - will double-check with @igdianov later
